segtypes/n64/asm.py

```diff
@@ -17,6 +17,10 @@ def get_file_header():
         ret.append(".set noreorder # don't insert nops after branches")
         ret.append(".set gp=64     # allow use of 64-bit general purpose registers")
         ret.append("")
+        preamble = options.get_generated_s_preamble()
+        if (preamble != ""):
```

Comment:
Sorry, just one more thing - to make this more pythonic, you don't need the parens around this condition, nor do you need to check if it's != "" - you can just do `if preamble` (here and in the psx one)

Reply:
Made these changes. 👍